python/tests/transformers/named_image_test.py

```diff
@@ -91,6 +91,15 @@ def setUpClass(cls):
         if(cls.numPartitionsOverride):
             cls.imageDf = cls.imageDF.coalesce(cls.numPartitionsOverride)
 
+    def __sortByFileOrder(self, ary):
```

Comment:
when do we use 1 _ vs 2 __ ?

Reply:
Hmm, I am not sure about DB convention, if you think I should use just 1 _ I can change it. In general 2 __ makes it semi private in python, meaning you need to try harder to access it, 1 _ has no special meaning.